### ISSUE TITLE
Improve usability and feedback of server URL entry

### ISSUE THREAD

**Issue by AndyDentFree** [realm/realm-browser-osx#269]
The Server URL field currently has a faint grey placeholder _realm://_ but if you quickly type into that field it vanishes and there is no longer any prompt to remind you that protocol must be used.

**There is no error message** if you enter a different protocol such as http - just the **Connect** button failing to be enabled.

As we have a default port 9080 used too, it makes sense to allow a simple IP address to be entered and then display that defaulted port as the user tabs out of the field.

Relying on placeholder text for important form information is highly discouraged, for example in [this article](https://uxplanet.org/mobile-form-usability-2279f672917d).

**Comment by einsteinx2** [comment]
+1 

I just wasted 10 minutes or so figuring this out today. Also if you try to connect directly to one of the other non-proxy ports it either crashes the app or just closes the window. First I had to figure out to type realm:// (which should be implied when nothing is entered, as well as you mentioned there's no error message for using http:// which I did), then I had to figure out to use the 9080 port after trying the other 2 and no port.

**Comment by einsteinx2** [comment]
Also worth noting that when reading the docs for the Object Server and seeing the usage of http to connect, I'm not quite sure why the Realm Browser even requires the use of realm:// anyway:

`let serverURL = URL(string: "http://my.realmServer.com:9080")!`

**Comment by einsteinx2** [comment]
Actually the issue I had earlier was connecting using the "Connect to Object Server" option, which I finally got to work.

For the life of me, I cannot get the Open button to enable when using the "Open Sync URL..." option. I have my correct realm:// url, and the correct username and password the works in my app, but I just can't get the Open button to enable. I've tried all kinds of other random things like using an email in the username field, trying various urls, etc, and nothing I do gets it to enable when using the Username option.
